### PR TITLE
[WPB-1226] Servantify internal Galley conversation endpoints

### PR DESCRIPTION
**Pull request by mdimjasevic**
The PR servantifies a few internal Galley endpoints.

Tracked by https://wearezeta.atlassian.net/browse/WPB-1226.

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/galley/src/Galley/API/Internal.hs [hunk 1]
```diff
@@ -117,11 +116,21 @@ internalAPI =
...
       <@> federationAPI
+      <@> conversationAPI
 
 federationAPI :: API IFederationAPI GalleyEffects
 federationAPI =
   mkNamedAPI @"get-federation-status" (const getFederationStatus)
 
+conversationAPI :: API IConversationAPI GalleyEffects
+conversationAPI =
+  mkNamedAPI @"conversation-channel" (const mempty)
```
Comment (akshaymankar) [inline services/galley/src/Galley/API/Internal.hs:127]: Such an absurd endpoint, I think it is not used. Let's just delete it?

**Review by akshaymankar — APPROVED** [review]
Looks good. That one weird endpoint can be deleted I think, feel free to do it in a separate PR if you like.

File: services/galley/src/Galley/API/Internal.hs (hunk 1 above)
Comment (mdimjasevic) [reply services/galley/src/Galley/API/Internal.hs]: Done in d94b80b0a1734be55c1616f94201aaeaedf82ea2.

**Review by akshaymankar — APPROVED** [review]
:shipit:

**Pull request merged**